testsuite/manualmode/src/test/java/org/wildfly/core/test/standalone/mgmt/ManagementInterfaceResourcesTestCase.java

```diff
@@ -144,10 +181,17 @@ private void runTest(int noRequestTimeout, ExceptionRunnable<Exception> test) th
             controller.reload();
 
             try (CLIWrapper cli = new CLIWrapper(true)) {
-                cli.sendLine(String.format("/system-property=%s:remove()", BACKLOG_PROPERTY));
-                cli.sendLine(String.format("/system-property=%s:remove()", CONNECTION_HIGH_WATER_PROPERTY));
-                cli.sendLine(String.format("/system-property=%s:remove()", CONNECTION_LOW_WATER_PROPERTY));
-                cli.sendLine(String.format("/system-property=%s:remove()", NO_REQUEST_TIMEOUT_PROPERTY));
+                if (useSystemProperties) {
+                    cli.sendLine(String.format(SYSTEM_PROPERTY_REMOVE, BACKLOG_PROPERTY));
+                    cli.sendLine(String.format(SYSTEM_PROPERTY_REMOVE, CONNECTION_HIGH_WATER_PROPERTY));
+                    cli.sendLine(String.format(SYSTEM_PROPERTY_REMOVE, CONNECTION_LOW_WATER_PROPERTY));
+                    cli.sendLine(String.format(SYSTEM_PROPERTY_REMOVE, NO_REQUEST_TIMEOUT_PROPERTY));
+                } else {
+                    cli.sendLine(String.format(HTTP_INTERFACE_UNDEFINE_ATTRIBUTE, BACKLOG_ATTRIBUTE));
```

Comment:
I think this needs handling similar to the L163/172 block. In default stability this will fail and the controller.stop will not be called.

Reply:
Ah good catch.

Reply:
@bstansberry actual if you scroll up slightly to line 166 I do call controller.stop() the moment the decision is made to not proceed with that test.

Reply:
Ok, and you return.